scripts/profile_boot.py

```
@@ -0,0 +1,143 @@
+        text = log_path.read_text(errors="replace")
+        n_lines = text.count("\n")
+        stamps = re.findall(r"\[\s*(\d+\.\d+)\]", text)
+        if stamps:
+            kernel_last = float(stamps[-1])
+    r["kernel_last_printk_s"] = kernel_last
+    r["log_lines"] = n_lines
+
+    try:
+        client.close()
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)